yarn.lock

```diff
@@ -5201,6 +5201,7 @@ node-releases@^1.1.8:
 node-sass@^4.12.0:
   version "4.12.0"
   resolved "https://registry.yarnpkg.com/node-sass/-/node-sass-4.12.0.tgz#0914f531932380114a30cc5fa4fa63233a25f017"
+  integrity sha512-A1Iv4oN+Iel6EPv77/HddXErL2a+gZ4uBeZUy+a8O35CFYTXhgA8MgLCWBtwpGZdCvTvQ9d+bQxX/QC36GDPpQ==
```

Comment:
This is happening bc the above `node-sass` change

Reply:
Just yarn uninstall node-sass? @thestephenmarshall